rs/nns/governance/canister/governance.did

```diff
@@ -107,6 +107,18 @@ type ClaimOrRefreshResponse = record {
   refreshed_neuron_id : opt NeuronId;
 };
 
+type RefreshVotingPower = record {
```

Comment:
This is the API that is just confirming the timer?
How about we call this "confirm following" which is the intent?

Reply:
Confirm following” sounds more straightforward to me, however, there could be cases where it might be confusing. If a neuron is already missing the voting reward, [its following on topics other than NeuronManagement is cleared](https://github.com/dfinity/ic/pull/2525/commits/7b96fcce7438f41c59315688e881ec0d47d7b7e7#diff-47cb67467f37606a03f7785a62befc6771b11fa71b463741d6fd826c21f05cb4R584-R592). This could be confusing because we’re confirming a cleared following list.

Reply:
> This is the API that is just confirming the timer?

Yes.

> How about we call this "confirm following" which is the intent?

This also has the effect of keeping your deciding voting power at 100% of potential voting power. Since your voting power is affected before your following, this maybe makes more sense. Also, this is consistent with the voting_power_refreshed_timestamp_seconds field, which we agreed to in an earlier PR.

Maksims makes an interesting point: once your following has been cleared, this doesn't restore your old following. At the same time, no matter what happens, this **always** restores your deciding voting power (back to 100% of potential voting power).

Reply:
I going to assume that we are ok with keeping the name RefreshVotingPower.